### ISSUE TITLE
[Question] How can I reproduce the safe driving results?

### ISSUE THREAD

**Issue by Gaiejj** [metadriverse/metadrive#687]
Thank you very much for your contributions to the Safe RL community! 
I am trying to reproduce the Safe RL baseline algorithms, namely `PPOLag` and `SACLag`, on MetaDrive.
<img width="582" alt="image" src="https://github.com/metadriverse/metadrive/assets/108712610/9e04616b-7cdb-4294-8fd3-928de89b6e79">
I found that the training success rate of `PPOLag` is roughly consistent with the paper, but the Cost is significantly higher. (Here is the performance of `PPOLag`. The Reward is the success rate.The dIfferent dolors denote different `num_scenarios`)
<img width="975" alt="image" src="https://github.com/metadriverse/metadrive/assets/108712610/5b7618d6-d61f-4e9d-9e54-b4951269cb4a">

I have read the paper and did not find any hyperparameters related to the environment settings. May I ask if you used the default hyperparameters provided in MetaDrive? Also, do you have any code references to reproduce the results in the paper? Thank you in advance!

**Comment by Gaiejj** [comment]
Thank you again for your answer! I no longer have any confusion.
